test/known_issues/test-dgram-bind-shared-ports-after-port-0.js

```diff
@@ -0,0 +1,71 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+'use strict';
+const common = require('../common');
+
+// This test should fail because at present `cluster` does not know how to share
+// a socket when `worker1` binds with `port: 0`, and others try to bind to the
+// assigned port number from `worker1`
+
+const assert = require('assert');
+const cluster = require('cluster');
+const dgram = require('dgram');
+const BYE = 'bye';
+
+if (cluster.isMaster) {
+  const worker1 = cluster.fork();
+  worker1.on('message', common.mustCall((msg) => {
+    const worker1Parts = msg.split(',');
+    assert.strictEqual(worker1Parts[0], 'success');
+    const PRT1 = Number(worker1Parts[1]);
+    assert.strictEqual(Number.isNaN(PRT1), false);
+
+    const worker2 = cluster.fork({ PRT1 });
+    worker2.on('message', common.mustCall());
+    worker2.on('exit', common.mustCall((code, signal) => {
+      assert.strictEqual(code, 0);
+      assert.strictEqual(signal, null);
+    }));
+    process.on('exit', () => {
+      worker1.send(BYE);
+      worker2.send(BYE);
+    });
+  }));
+  worker1.on('exit', common.mustCall((code, signal) => {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+  }));
+} else {
+  process.on('message', common.mustCall((msg) => {
+    assert.strictEqual(msg, BYE);
+    process.exit(0);
+  }));
+
+  const PRT1 = process.env.PRT1 || 0;
+  const socket1 = dgram.createSocket('udp4', common.mustCall());
+  const opts = { address: common.localhostIPv4, port: PRT1, exclusive: false };
+  socket1.bind(opts, common.mustCall(() => {
+    const port1 = socket1.address().port;
+    process.send(`success,${port1}`);
```

Comment:
You can send a full JS object here, let's do that as it will make the test easier to read.